modules/ldsg.js

```diff
@@ -65,6 +65,16 @@ async function processCardAction(int) {
           .setPlaceholder("Your reply to the user")
```

Comment:
it wont let me comment earlier, but `const modal` should have its name changed since there are multiple modals now